src/Entity/IdeasWorkshop/Question.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityNameTrait;
+use AppBundle\Entity\EntityPositionTrait;
+use AppBundle\Entity\EntityPublishableTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Question
+{
+    use EntityNameTrait;
+    use EntityPublishableTrait;
+    use EntityPositionTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
```

Comment:
Same here, you can use directly : `@ORM\GeneratedValue`

Reply:
:heavy_check_mark: